rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -2341,4 +2341,23 @@ boolean checkJobPermissionMethod(@HeaderParam("sessionid") String sessionId, @Pa
     @Produces(MediaType.APPLICATION_JSON)
     Set<String> addJobSignal(@HeaderParam("sessionid") String sessionId, @QueryParam("signal") String signal,
             @PathParam("jobid") String jobId) throws RestException;
+
+    /**
+     *
+     *  Check if the user has the permission to execute the methods passed as argument for each the given job ids
+     *
+     * @param sessionId
+     *            current session
+     * @param multipart
+     *            a form with the jobIds list and methods list as form data
+     *
+     * @return a map containing pairs that represent the job id as key and a map as value containing the method name
+     * and true/false if the user has permission for the jobId and method
+     */
+    @POST
+    @Path("jobs/permission/methods")
+    @Consumes(MediaType.APPLICATION_JSON)
```

Comment:
Shouldn't it consume `MediaType.MULTIPART_FORM_DATA` ?

Reply:
the input will be in json format so we need MediaType.APPLICATION_JSON here